services/QuillLMS/client/app/bundles/Staff/components/styleGuide/__tests__/dataTable.test.jsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
```

Comment:
I'm still appreciating the approach in `@testing-library`.  I find it a lot easier to understand what the test is trying to check for.